x-pack/plugins/maps/public/routing/routes/list/maps_list_view.tsx

```diff
@@ -135,10 +164,13 @@ export class MapsListView extends React.Component {
     this.setState({ showDeleteModal: true });
   };
 
-  onTableChange = ({ page, sort = {} }) => {
-    const { index: pageIndex, size: pageSize } = page;
+  onTableChange = ({
+    page,
+    sort,
+  }: Criteria<SelectionItem> | CriteriaWithPagination<SelectionItem>) => {
```

Comment:
Can type every be `Criteria<SelectionItem>`? How about just 

```
onTableChange = ({
    const { index: pageIndex, size: pageSize } = page;	    page,
    sort,
  }: CriteriaWithPagination<SelectionItem>) => {
```

Reply:
Fixed this.